pdns/dnsdist-protobuf.cc

```diff
@@ -186,6 +192,134 @@ void DNSDistProtoBufMessage::serialize(std::string& data) const
   }
 
   m.commitResponse();
+
+  for (const auto& [key, value] : d_metaTags) {
```

Comment:
Add metas *before* commiting ?

Reply:
So that's what I initially did, but that doesn't work because the meta tags are not in the response, so we would be trying to add fields after an incomplete field (the message) :-)